tests/unit/amo/pages/TestCategory.js

```diff
@@ -434,18 +433,13 @@ describe(__filename, () => {
     });
   });
 
-  it('renders the themes shelves with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
+  it('renders the themes shelves with the ADDON_TYPE_THEMES_FILTER filter', () => {
```

Comment:
Before I believe we added this test just b/c of the config now I am wondering if we need this..? See [PR/commit](https://github.com/mozilla/addons-frontend/commit/cb33d11172242d4a1146c79edad9a6e2a5e2bfe1#diff-6077ad35fee63ec7e99470d3e202d584R441) that added this. 

if we do keep it I would adjust the name slightly and should we add to the end ~ "...if it's a theme type" and consider adding an extensions test too b/c if we have this for theme should we add something for extensions? I feel like that's what we usually do to show the different types, right?

note: I have similar comment below